Makefile

```diff
@@ -10,10 +10,17 @@ fmt:
 		exit 1; \
 	fi
 
+SAMPLEDATA:=../zq-sample-data/README.md
+
+$(SAMPLEDATA):
+	git clone https://github.com/mccanne/zq-sample-data $(@D)
```

Comment:
If we're going to pull the sample data here, I'd prefer it get pulled into a directory within the developer's `zq` directory; I think this is pulling to the directory above?

Consider also adding a `--depth 1` here to make this a shallow clone.

Reply:
I threw this up so it'd work in CI. I have some questions about our approach for this

1. Should we use git submodules instead of a Makefile dependency?
2. If not submodules, should these tests be part of `tests-system` as I've done, or should they be part of a different target? Ideas include: `test-docs` (isolating; not my preference) or `tests-heavy` (a bit forward thinking; indicating possibly longer-running tests and larger test data sets)?

Reply:
Offline conversation summary: no submodules because we want zq to be easy to clone, which includes room for use as a package in another Go project.

Reply:
Done